src/korean.js

```diff
@@ -59,6 +59,9 @@ class Korean {
     if (word[wordLength - 2] === '하') {
       conjugate = word.slice(0, wordLength - 2);
       return conjugate.concat('해');
+    } else if (word[wordLength - 2] === '앉' || word[wordLength - 2] === '솟' || word[wordLength - 2] === '닫') {
```

Comment:
@songz @thisisrai @minsooshin - finally got around to fixing it and opened PR #85 . Thanks again for all the help!